FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+  title: "How satisfied are you with file uploads?"
+
+  position: POSITION_CONFIG, // See positioning section
+  translations: TRANSLATION_KEYS // See internationalization section
+
+  // Trigger side effects with the collected data,
+  // for example posting it back into your own CRM
+  onAfterSubmit: (feedback) => {
+    storeFeedbackInCRM({
+      score: feedback.score,
+      comment: feedback.comment
+    })
+  }
+})
+```
+
+See also:
+
+- [Positioning and behavior](#positioning-and-behavior) for the position option.
+- [Runtime language configuration](#runtime-language-configuration) if you want to translate the feedback UI.
+
+## Automated feedback collection
+
+When automated feedback collection is enabled the Bucket SDK will stay in contact with the Bucket service. When a user triggers an event used in a feature, and the user is eligible for being prompted for feedback, the Bucket service will push an event to the SDK instance. By default, this event will open up the Bucket feedback UI, but you have some abilities to intercept this event and add your own overrides to it.
+
+### Initializing automated feedback collection
+
+You can enable automated collection in the `bucket.init()`-call:
+
+```javascript
+bucke.init({
+  feedback: {
+    automaticPrompting: true,
+  },
+});
+```
+
+### Overriding prompt event defaults
+
+If you are not satisfied with the default UI behavior when an automated prompt event arrives, you can intercept and override settings like this:
+
+```javascript
+bucket.init({
+  feedback: {
+    automaticPrompting: true,
+    promptHandler: (promptMessage, handlers) => {
+      // Pass your overrides here. Everything is optional
+      handlers.openFeedbackForm({
+        title: promptMessage.question,
+
+        position: POSITION_CONFIG, // See positioning section
+        translations: TRANSLATION_KEYS, // See internationalization section
+
+        // Trigger side effects with the collected data,
+        // for example posting it back into your own CRM
+        onAfterSubmit: (feedback) => {
+          storeFeedbackInCRM({
+            score: feedback.score,
+            comment: feedback.comment,
+          });
+        },
+      });
+    },
+  },
+});
+```
+
+See also:
+
+- [Positioning and behavior](#positioning-and-behavior) for the position option.
+- [Runtime language configuration](#runtime-language-configuration) if you want to translate the feedback UI.
+- [Use your own UI to collect feedback](#using-your-own-ui-to-collect-feedback) if the feedback UI doesn't match your design.
+
+## Positioning and behavior
+
+The feedback UI can be configured to be placed and behave in 3 different ways:
+
+### Positioning configuration
+
+#### Modal
+
+A modal overlay with a backdrop that blocks interaction with the remaining page. Can be dismissed with `<ESC>` or a close button. It is always centered on the page, making it the primary thing the user needs to focus on.
```

Comment:
I suggest:

A modal overlay with a backdrop that blocks interaction with the underlying page. It can be dismissed with the keyboard shortcut `<ESC>` or the dedicated close button in the top right corner. It is always centered on the page, capturing focus, and making it the primary interface the user needs to interact with.

Reply:
```suggestion
A modal overlay with a backdrop that blocks interaction with the underlying page. It can be dismissed with the keyboard shortcut `<ESC>` or the dedicated close button in the top right corner. It is always centered on the page, capturing focus, and making it the primary interface the user needs to interact with.
```